packages/shorebird_cli/pubspec.yaml

```diff
@@ -37,6 +37,8 @@ dependencies:
     path: ../scoped
   shorebird_code_push_client:
     path: ../shorebird_code_push_client
+  shorebird_code_push_protocol:
+    path: ../shorebird_code_push_protocol
```

Comment:
Why is this the first time we've depended on this?

Reply:
Answered offline – this contains the `AuthProvider` enum, which contains the list of supported SSO providers